apps/web/public/service-worker.js

```diff
@@ -11,43 +13,82 @@ self.addEventListener("push", async (event) => {
     return;
   }
 
-  const title = notificationData.title || "You have a new notification from Cal.com";
-  const image = "https://cal.com/api/logo?type=icon";
-  const newNotificationOptions = {
-    requireInteraction: true,
-    ...notificationData,
-    icon: image,
-    badge: image,
-    data: {
-      url: notificationData.data?.url || "https://app.cal.com",
-    },
-    silent: false,
-    vibrate: [300, 100, 400],
-    tag: `notification-${Date.now()}-${Math.random()}`,
-  };
-
-  const existingNotifications = await self.registration.getNotifications();
-
-  // Display each existing notification again to make sure old ones can still be clicked
-  existingNotifications.forEach((notification) => {
-    const options = {
-      body: notification.body,
-      icon: notification.icon,
-      badge: notification.badge,
-      data: notification.data,
-      silent: notification.silent,
-      vibrate: notification.vibrate,
-      requireInteraction: notification.requireInteraction,
-      tag: notification.tag,
-    };
-    self.registration.showNotification(notification.title, options);
-  });
+  const title = notificationData.title || "New Cal.com Notification";
```

Comment:
The service worker would trigger notification and send message to play sound if notification type is instant meetings